### DISCUSSION TITLE
Customizing console logs

### DISCUSSION THREAD

**Discussion by avin-kavish** [fastify/fastify#4026] [Q&A] [answered]
I would like to customize this log line,
```
[11:55:06.743] INFO: incoming request
```

Ideally I would like to have this,

```
11:55:06.743 - info - GET /my/route/
```

**Comment by mcollina** [comment, marked as answer]
I've create an issue from this: https://github.com/fastify/fastify/issues/4027. We should definitely provide a better DX.

**Reply by Uzlopak** [reply]
We have one-line-logger
